examples/docker-compose-proxy/verify.sh

```
@@ -0,0 +1,129 @@
+
+cleanup() {
+  (cd "$EXAMPLE_DIR" && docker compose down -v >/dev/null 2>&1) || true
+}
+trap cleanup EXIT
+
+mkdir -p "$WORK"
+install -m 600 "$CONFIG_SRC" "$CONFIG"
+
+wait_for_health() {
+  local i
+  for i in $(seq 1 80); do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)